purchases/src/test/java/com/revenuecat/purchases/google/BillingWrapperTest.kt

```diff
@@ -2773,7 +2776,7 @@ class BillingWrapperTest {
             }
         )
 
-        lock.await()
+        lock.await(2, TimeUnit.SECONDS)
```

Comment:
It would probably be fine here with 200-300ms... But I think it's ok. If it's not done in 300ms, it shouldn't be done in 2s either, so this should fail as well.

Reply:
yes, makes sense. I will update!